src/SPC/builder/macos/MacOSBuilder.php

```diff
@@ -97,9 +97,17 @@ public function buildPHP(int $build_target = BUILD_TARGET_NONE): void
         $this->emitPatchPoint('before-php-configure');
         SourcePatcher::patchBeforeConfigure($this);
 
-        $json_74 = $this->getPHPVersionID() < 80000 ? '--enable-json ' : '';
+        $phpVersionID = $this->getPHPVersionID();
+        $json_74 = $phpVersionID < 80000 ? '--enable-json ' : '';
         $zts = $this->getOption('enable-zts', false) ? '--enable-zts --disable-zend-signals ' : '';
 
+        $opcache_jit = !$this->getOption('disable-opcache-jit', false);
+        if ($opcache_jit) {
+            $opcache_jit = $phpVersionID >= 80500 ? '--enable-opcache-jit ' : '';
+        } else {
+            $opcache_jit = '--disable-opcache-jit ';
+        }
```

Comment:
Revert this please, otherwise we take the ability to compile opcache jit for php < 8.5.

Reply:
The previous opcache jit option we used (enable is `''`) is working well, but explictly passing it is more clearer.